lib/tasks/email_checkin_reminder.rake

```diff
@@ -0,0 +1,17 @@
+desc 'Send email reminder about upcoming checkins'
+task email_checkin_reminder: :environment do
+  if AppConfig.first.upcoming_checkin_email_active?
+    # get all reservations that end today and aren't already checked in
+    upcoming_reservations = Reservation.due_soon
+    Rails.logger.info "Found #{upcoming_reservations.size} reservations due "\
+                      'for checkin. Sending reminder emails...'
+    upcoming_reservations.each do |upcoming_reservation|
+      UserMailer.reservation_status_update(upcoming_reservation).deliver
+    end
+    Rails.logger.info 'Done!'
+  else
+    Rails.logger.info 'Upcoming check in emails are not sent by admin. '\
```

Comment:
Let's tweak this copy - `Reservations is not configure to send upcoming check-in e-mails.` and continue from there